backend/resources/contact.py

```diff
@@ -33,13 +33,13 @@ class SendMessage(Resource):
     """
 
     external_message_subject = """
-        [CfP] Witaj!
+        [Code fof Poznań] Witaj na pokładzie!
     """
 
     external_message_body = """
-        Cześć,
+        Cześć!
         
-        tu załoga Code For Poznań! Cieszymy się, że z nami jesteś!
+        Tu załoga Code For Poznań! Chcieliśmy dać znać, że otrzymaliśmy Twoją wiadomość i postaramy się jak najszybciej na nią odpowiedzieć.
```

Comment:
Chcielibyśmy

Reply:
A jakieś zamknięcie tego mejla tu nie powinno być dodane? Ja bym to osobiście jakoś tak widziała:

> Cześć,
> 
> Dziękujemy za Twoją wiadomość! Prześlemy Ci odpowiedź najszybciej, jak to będzie możliwe.
> 
> Pozdrawiamy,
> załoga Code for Poznań